src/common/offline_transformations/src/pruning/shrink_weights.cpp

```diff
@@ -13,28 +13,73 @@
 #include <ngraph/log.hpp>
 #include <ngraph/ngraph.hpp>
 
+template<typename T>
+static std::string vec_to_str(const std::vector<T> m) {
+    std::ostringstream out;
+    out << "[ ";
+    for (auto & val : m)
+        out << val << ' ';
+    out << "]";
+    return out.str();
+}
```

Comment:
Just to be sure everything's cool 
```suggestion
static std::string vec_to_str(const std::vector<T>& m) {
    std::ostringstream out;
    out << "[ ";
    for (const auto & val : m)
        out << val << ' ';
    out << "]";
    return out.str();
}
```

Reply:
Thanks! Done